firebase.json

```
@@ -28,7 +28,7 @@
 
       { "source": "/accessibility", "destination": "/ui/accessibility-and-internationalization/accessibility", "type": 301 },
       { "source": "/adaptations", "destination": "/platform-integration/platform-adaptations", "type": 301 },
-      { "source": "/adaptive*", "destination": "/ui/layout/responsive/adaptive-responsive", "type": 301 },
+      { "source": "/adaptive*", "destination": "/ui/adaptive-responsive/overview", "type": 301 },
```

Since you got rid of the index in favor of the index page:

```suggestion
      { "source": "/adaptive*", "destination": "/ui/adaptive-responsive", "type": 301 },
```